src/main/java/org/ballerinax/datamapper/DataMapperPlugin.java

```diff
@@ -505,7 +525,7 @@ private void extractFunctionCalls(BLangTypeDefinition typeDefinition) throws IOE
             }
         }
 
-        if(functions.length() != 0) {
+        if (functions.length() != 0) {
```

Comment:
if (!functions.isEmpty)

Reply:
I could not find property isEmpty in StringBuilder class. Hence I left this as it is.

Reply:
ack